# don't add architecture as suffix. Just makes install scripts complicated.
Index: warsow-2.0/source/source/CMakeLists.txt
===================================================================
--- warsow-2.0.orig/source/source/CMakeLists.txt	2015-11-28 18:22:04.000000000 +0100
+++ warsow-2.0/source/source/CMakeLists.txt	2015-12-12 02:03:24.884177436 +0100
@@ -249,7 +249,6 @@
     set(ZLIB_LIBRARY "")
     add_definitions(-DZLIB_RUNTIME)
 
-    set(CMAKE_EXECUTABLE_SUFFIX ".${CMAKE_SYSTEM_PROCESSOR}${CMAKE_EXECUTABLE_SUFFIX}")
     set(CMAKE_SHARED_LIBRARY_SUFFIX "_${CMAKE_SYSTEM_PROCESSOR}${CMAKE_SHARED_LIBRARY_SUFFIX}")
 endif()
 
